src/Microsoft.DocAsCode.AzureMarkdownRewriters/AzureBlockRules/AzureMigrationHtmlMetadataBlockRule.cs

```diff
@@ -82,7 +82,13 @@ private AzureHtmlMetadata GetAttributesFromHtmlContent(string htmlContent)
                 {
                     if (string.Equals(attribute.Name, "pageTitle", StringComparison.OrdinalIgnoreCase))
                     {
-                        attributes["title"] = attribute.Value;
+                        // Per Azure's request, migration script should convert "| Microsoft Azure" to "| Microsoft Docs" in title.
+                        var title = attribute.Value;
```

Comment:
No, TestAzureMarkdownRewriters_AzureProperties only test AzureHtmlMetadataBlockRule 

---
In reply to: [85067711](https://github.com/dotnet/docfx/pull/826#discussion_r85067711) [](ancestors = 85067711)